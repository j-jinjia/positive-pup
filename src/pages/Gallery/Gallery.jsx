import Layout from "../../components/Layout/Layout";

const Gallery = () => {
  return (
    <Layout>
      <h1>Gallery</h1>
    </Layout>
  );
};

export default Gallery;
