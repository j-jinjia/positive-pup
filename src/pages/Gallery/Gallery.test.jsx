import { customRender } from "../../utils/testUtils";
import Gallery from "./Gallery";

it("should render the Gallery page", () => {
  const { container } = customRender(<Gallery />);

  expect(container).toMatchSnapshot();
});
