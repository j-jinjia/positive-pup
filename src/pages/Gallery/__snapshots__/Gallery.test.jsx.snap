// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Gallery page 1`] = `
<div>
  <h1>
    Gallery
  </h1>
  <div
    class="footer"
  >
    <div
      class="footer__image-container"
    >
      <img
        alt="logo"
        class="footer__logo"
        src="footer-logo.svg"
      />
    </div>
    <ul
      class="footer__links-container"
    >
      <li
        class="footer__list-item"
      >
        <a
          class="footer__link"
          href="/about"
        >
          About
        </a>
      </li>
      <li
        class="footer__list-item"
      >
        <a
          class="footer__link"
          href="/services"
        >
          Services
        </a>
      </li>
      <li
        class="footer__list-item"
      >
        <a
          class="footer__link"
          href="/gallery"
        >
          Gallery
        </a>
      </li>
      <li
        class="footer__list-item"
      >
        <a
          class="footer__link"
          href="/terms-and-conditions"
        >
          Terms and conditions
        </a>
      </li>
      <li
        class="footer__list-item"
      >
        <a
          class="footer__link"
          href="/faqs"
        >
          FAQs
        </a>
      </li>
      <li
        class="footer__list-item"
      >
        <a
          class="footer__link"
          href="/get-in-touch"
        >
          Contact
        </a>
      </li>
    </ul>
    <hr />
    <div
      class="footer__secondary-links-container"
    >
      <div
        class="footer__copyright-text"
      >
        © 2022 The Positive Pup. All rights reserved.
      </div>
      <div
        class="footer__secondary-links-container--social-links"
      >
        <a
          href="https://www.facebook.com/thepositivepupdogtraining"
          target="blank"
        >
          <img
            alt="link to facebook"
            src="facebook-logo.svg"
          />
        </a>
        <a
          href="https://www.instagram.com/the_positivepup/"
          target="blank"
        >
          <img
            alt="link to instagram"
            src="instagram-logo.svg"
          />
        </a>
      </div>
    </div>
  </div>
</div>
`;

exports[`should render the footer component 1`] = `
<div>
  <h1>
    Gallery
  </h1>
  <div
    class="footer"
  >
    <div
      class="footer__image-container"
    >
      <img
        alt="logo"
        class="footer__image-container--logo"
        src="footer-logo.svg"
      />
    </div>
    <ul
      class="footer__links-container"
    >
      <li
        class="footer__list-item"
      >
        <a
          class="footer__links"
          href="/about"
        >
          About
        </a>
      </li>
      <li
        class="footer__list-item"
      >
        <a
          class="footer__links"
          href="/services"
        >
          Services
        </a>
      </li>
      <li
        class="footer__list-item"
      >
        <a
          class="footer__links"
          href="/Gallery"
        >
          Gallery
        </a>
      </li>
      <li
        class="footer__list-item"
      >
        <a
          class="footer__links"
          href="/terms-and-conditions"
        >
          Terms and conditions
        </a>
      </li>
      <li
        class="footer__list-item"
      >
        <a
          class="footer__links"
          href="/faqs"
        >
          FAQs
        </a>
      </li>
      <li
        class="footer__list-item"
      >
        <a
          class="footer__links"
          href="/get-in-touch"
        >
          Contact
        </a>
      </li>
    </ul>
    <hr />
    <div
      class="footer__secondary-links-container"
    >
      <div
        class="footer__secondary-links-container--copyright"
      >
        © 2022 The Positive Pup. All rights reserved.
      </div>
      <div
        class="footer__secondary-links-container--social-links"
      >
        <a
          href="https://www.facebook.com/thepositivepupdogtraining"
          target="blank"
        >
          <img
            alt="link to facebook"
            src="facebook-logo.svg"
          />
        </a>
        <a
          href="https://www.instagram.com/the_positivepup/"
          target="blank"
        >
          <img
            alt="link to instagram"
            src="instagram-logo.svg"
          />
        </a>
      </div>
    </div>
  </div>
</div>
`;
